src/main/java/org/opensearch/sdk/sample/helloworld/rest/RestHelloAction.java

```diff
@@ -37,9 +38,13 @@ public List<Route> routes() {
     }
 
     @Override
-    public ExtensionRestResponse handleRequest(Method method, String uri) {
+    public ExtensionRestResponse handleRequest(ExtensionRestRequest request) {
         // We need to track which parameters are consumed to pass back to OpenSearch
         List<String> consumedParams = new ArrayList<>();
+        Method method = request.method();
+        String uri = request.uri();
+        // TODO: should we extract token and then add it to the list of consumed parameters?
```

Comment:
I think these are params from the client (named wildcards or REST parameters). You're adding the token separately.  So I think no.

Reply:
okay... i will remove it